server/app-engine/static/graas.js

```diff
@@ -471,18 +493,22 @@ function handleOkay() {
     p.style.background = "green";
     var p = document.getElementById(BUS_SELECT_DROPDOWN);
     vehicleID = p.value
+
+    document.cookie = `${VEHICLE_ID_COOKIE_NAME}=${vehicleID}; max-age=${MAX_VEHICLE_ID_AGE_SECS}`;
     util.log("- vehicleID: " + vehicleID);
 
-    p = document.getElementById(TRIP_SELECT_DROPDOWN);
-    var entry = tripIDLookup[p.value];
+    if(!useBulkAssignmentMode){
+        p = document.getElementById(TRIP_SELECT_DROPDOWN);
+        var entry = tripIDLookup[p.value];
 
-    if (isObject(entry)) {
-        tripID = entry['trip_id'];
-    } else {
-        tripID = entry;
-    }
+        if (isObject(entry)) {
```

Comment:
looks like you have tabs or other extra indents here

Reply:
don't think so, might be missing something